projects/step-frontend/src/lib/modules/timeseries/pages/dashboard/chart-dashlet/settings/chart-dashlet-settings.component.ts

```diff
@@ -23,20 +23,25 @@ export class ChartDashletSettingsComponent implements OnInit {
   private _inputData: ChartDashletSettingsData = inject<ChartDashletSettingsData>(MAT_DIALOG_DATA);
   private _formBuilder = inject(FormBuilder);
   private _dialogRef = inject(MatDialogRef);
+
+  _attributesByKey: Record<string, MetricAttribute> = {};
```

Comment:
Since it's a simple field and not injection it shouldn't started from underscore.
Probably it can be marked as private
```suggestion
  attributesByKey: Record<string, MetricAttribute> = {};
```